src/harness/sas.py

```diff
@@ -101,8 +103,8 @@ def _CbsdRequest(self, method_name, request, ssl_cert=None, ssl_key=None):
     return RequestPost('https://%s/%s/%s' % (self.cbsd_sas_active_base_url, self.cbsd_sas_version,
                                              method_name), request,
                        self._tls_config.WithClientCertificate(
-                           ssl_cert or GetDefaultCbsdSSLCertPath(),
-                           ssl_key or GetDefaultCbsdSSLKeyPath()))
+                           ssl_cert or GetDefaultDomainProxySSLCertPath(),
+                           ssl_key or GetDefaultDomainProxySSLKeyPath()))
```

Comment:
GetDefaultDomainProxySSLCertPath() use domain_proxy(cert/key).
I think you need to define those two methods
def GetDefaultCbsdSSLCertPath():
  return os.path.join('certs', 'device_a.cert')

def GetDefaultCbsdSSLCertPath():
  return os.path.join('certs', 'device_a.key')